sdcm/cluster_aws.py

```diff
@@ -351,6 +351,21 @@ def network_config_ipv6_workaround_script():  # pylint: disable=invalid-name
             IPv6_CIDR=`curl -s ${BASE_EC2_NETWORK_URL}${MAC}/subnet-ipv6-cidr-blocks`
 
             grep -qi "amazon linux" /etc/os-release || sudo ip route add $IPv6_CIDR dev eth0
+
+            grep -q IPV6_AUTOCONF /etc/sysconfig/network-scripts/ifcfg-eth0
+            if [[ $? -eq 0 ]]; then
+                sudo sed -i 's/^IPV6_AUTOCONF=[^ ]*/IPV6_AUTOCONF=yes/' /etc/sysconfig/network-scripts/ifcfg-eth0
```

Comment:
@juliayakovlev `'s/^IPV6_AUTOCONF=[^ ]*$` ?

Reply:
It search for line, started from"IPV6_AUTOCONF=" and any other substring after "=".
It's works, I tested.
Do you think about some problem?